scripts/proxy/summarize_session.py

```
@@ -0,0 +1,1198 @@
+
+    seq = int(match.group(1))
+    time_str = match.group(2)
+    msg_count = int(match.group(3))
+    preview = match.group(4).replace("_", " ")
+
+    # Parse time (assume today's date for grouping purposes)
+    try:
+        hour = int(time_str[0:2])
+        minute = int(time_str[2:4])
+        second = int(time_str[4:6])
+        timestamp = datetime.now().replace(hour=hour, minute=minute, second=second, microsecond=0)
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)